src/screens/TestCases/Modal/CustomFilterModal.js

```diff
@@ -88,9 +139,63 @@ const CustomFilterModal = ({
           }) => (
             <div className="custom-toolkit" {...props}>
```

Comment:
why spread props  here

Reply:
its help to open the text filter and number filter modal